Subject: Fix library SONAME
Author: Julien Lamy <lamy@unistra.fr>
Forwarded: https://github.com/lamyj/dcmtkpp/commit/0a7b8b9eb65be99b20b08f3b1b51b96da2dbcdf5
--- a/tests/code/Exception.cpp
+++ b/tests/code/Exception.cpp
@@ -31,6 +31,6 @@ BOOST_AUTO_TEST_CASE(ConditionConstructor)
 {
     dcmtkpp::Exception const exception(EC_IllegalParameter);
     BOOST_CHECK_EQUAL(exception.get_source(), dcmtkpp::Exception::Source::Condition);
-    BOOST_CHECK_EQUAL(exception.what(), EC_IllegalParameter.text());
+    BOOST_CHECK_EQUAL(exception.what(), OFCondition(EC_IllegalParameter).text());
     BOOST_CHECK(exception.get_condition() == EC_IllegalParameter);
 }

