Subject: Fix library SONAME
Author: Julien Lamy <lamy@unistra.fr>
Forwarded: https://github.com/lamyj/dcmtkpp/commit/3e56d7494f1856bd0c8cb9694919f5e99e446036
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 2.8)
+
 project("dcmtkpp")
+set(dcmtkpp_MAJOR_VERSION 0)
+set(dcmtkpp_MINOR_VERSION 2)
+set(dcmtkpp_PATCH_VERSION 1)
+set(dcmtkpp_VERSION 
+    ${dcmtkpp_MAJOR_VERSION}.${dcmtkpp_MINOR_VERSION}.${dcmtkpp_PATCH_VERSION})
 
 option(BUILD_EXAMPLES "Build the examples directory." ON)
 
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -17,5 +17,7 @@
 install(FILES ${headers} ${templates} DESTINATION include/dcmtkpp)
 
 target_link_libraries(dcmtkpp ${DCMTK_LIBRARIES} jsoncpp ${ICU_LIBRARIES})
-
+set_target_properties(dcmtkpp PROPERTIES 
+    VERSION ${dcmtkpp_VERSION}
+    SOVERSION ${dcmtkpp_MAJOR_VERSION})
 
